### PR TITLE
Refactor koa

### PR DESCRIPTION
**Pull request by maximelkin**

### CODE REVIEW

File: frameworks/JavaScript/koa/benchmark_config.json [hunk 1]
```diff
@@ -1,70 +1,71 @@
...
+        "orm": "Raw",
+        "platform": "None",
+        "webserver": "None",
+        "os": "Linux",
+        "database_os": "Linux",
+        "display_name": "koa",
+        "notes": "",
+        "versus": "nodejs"
+      }
+      ]
```
Comment (nbrady-techempower) [inline frameworks/JavaScript/koa/benchmark_config.json]: missing a `}` before `]`

File: frameworks/JavaScript/koa/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (nbrady-techempower) [inline frameworks/JavaScript/koa/.gitignore]: you wont need gitignores anymore since the test will be built and run in a docker container

**Comment by maximelkin** [comment]
Something wrong with builds?

**Comment by nbrady-techempower** [comment]
@maximelkin I restarted the build for you. For the postgres tests, you'll want to use `randomnumber` vs `randomNumber` We'll have to make a note of that in the docs. https://api.travis-ci.org/v3/job/357754913/log.txt

**Comment by maximelkin** [comment]
@nbrady-techempower Thank you.
Also add note what mongo collection name should be lowercase.

**Pull request merged**